metadata-jobs/mce-consumer/src/main/java/com/linkedin/metadata/kafka/MetadataChangeEventsProcessor.java

```diff
@@ -52,7 +52,7 @@ public void consume(final ConsumerRecord<String, GenericRecord> consumerRecord)
 
     try {
       event = EventUtils.avroToPegasusMCE(record);
-      log.debug("MetadataChangeEvent {}", event);
```

Comment:
let's do this as a metrics report to indicate aliveness, instead of logging each event.

Reply:
are you suggesting we create an endpoint to retrieve these metrics?

Reply:
Metrics -> one reporter can be a periodic logger to this same log file, another reporter can be connected up with JMX / prometheus etc.